dll/appcompat/apphelp/sdbapi.c

```diff
@@ -534,6 +534,116 @@ TAGREF WINAPI SdbGetLayerTagRef(HSDB hsdb, LPCWSTR layerName)
 }
 
 
+#ifndef REG_SZ
+#define REG_SZ 1
+#define REG_DWORD 4
+#define REG_QWORD 11
+#endif
+
+
+/**
+ * Retrieve a Data entry
+ *
+ * @param [in]  pdb                     The database.
+ * @param [in]  tiExe                   The tagID to start at
+ * @param [in,opt]  lpszDataName        The name of the Data entry to find, or NULL to return all.
+ * @param [out,opt]  lpdwDataType       Any of REG_SZ, REG_QWORD, REG_DWORD, ...
+ * @param [out]  lpBuffer               The output buffer
+ * @param [in,out,opt]  lpcbBufferSize  The size of lpBuffer in bytes
+ * @param [out,opt]  ptiData            The tagID of the data
+ *
+ * @return  ERROR_SUCCESS
+ */
+DWORD WINAPI SdbQueryDataExTagID(PDB pdb, TAGID tiExe, LPCWSTR lpszDataName, LPDWORD lpdwDataType, LPVOID lpBuffer, LPDWORD lpcbBufferSize, TAGID *ptiData)
+{
+    TAGID tiData, tiValueType, tiValue;
+    DWORD dwDataType, dwSizeRequired, dwInputSize;
+    LPCWSTR lpStringData;
+    /* Not supported yet */
+    if (!lpszDataName)
```

Comment:
MSDN documents both a NULL string, and a string with delimiters in them.
This can be added later if we need it for something